srcpkgs/lapce/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'lapce'
+pkgname=lapce
+version=0.2.0
+revision=1
+build_style=cargo
+hostmakedepends="cargo python cmake pkg-config"
```

Comment:
does this work with python3? new python 2 packages will not be accepted

Reply:
The official rpm package builds without python at all (even though the arch package has it as dependency), so I'll give that a go.